test/fixtures/parser/ansi/transactions.yml

```diff
@@ -1,31 +1,31 @@
-file:

```

Comment:
These diffs are weird, too. Maybe different versions of the YAML package default to different indentation?